lib/search.js

```diff
@@ -21,7 +21,7 @@ module.exports = function(appConfig, date_field) {
         return true;
     }
 
-    function performSearch(context, req, res, config) {
+    function performSearch(context, req, res, config, fn) {
```

Comment:
It would be better if this were set in the endpoint `config`. 

`config.post_process = function(data) {...}`